go/mysql/replication/filepos_gtid.go

```diff
@@ -147,6 +156,13 @@ func (gtid FilePosGTID) Union(other GTIDSet) GTIDSet {
 	return filePosOther
 }
 
+// Union implements GTIDSet.Union().
+func (gtid FilePosGTID) InPlaceUnion(other GTIDSet) GTIDSet {
```

Comment:
Should this be called `UnionInPlace` to match `AddGTIDInPlace`. Also, I just noticed `FilePosGTID.AddGTIDInPlace`  is a super verbose function name.

Reply:
Renamed as `UnionInPlace`. Agreed that `AddGTIDInPlace` is very verbose; I was contemplating whether we should just add a `inPlace bool` argument to `AddGTID` but decided agaist so as not to affect so many files.